phpseclib/Net/SshMsgCustom.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace phpseclib3\Net;
+
+/**
+ * @internal
+ */
+abstract class SshMsgCustom
```

Comment:
I think it'd be better if this were named Extra, instead of Custom.

Reply:
Done